scheduler/repositories/sqlalchemy/task_store.py

```diff
@@ -19,17 +20,38 @@ def __init__(self, datastore: SQLAlchemy) -> None:
         self.datastore = datastore
 
     def get_tasks(
-        self, scheduler_id: Union[str, None], status: Union[str, None], offset: int = 0, limit: int = 100
+        self,
+        scheduler_id: Union[str, None],
+        type: Union[str, None],
+        status: Union[str, None],
```

Comment:
`Optional[str]`

Reply:
Read somewhere that Union was the way to go, but forgot where I read that or if it's correct I don't know. Looked it up: https://mypy.readthedocs.io/en/stable/getting_started.html#additional-types-and-the-typing-module

> You can again use [Union](https://docs.python.org/3/library/typing.html#typing.Union) and use Union[str, None] – or alternatively, use the type Optional[str]. These two types are identical and interchangeable: Optional[str] is just a shorthand or alias for Union[str, None]. It exists mostly as a convenience to help function signatures look a little cleaner:

`Optional` it is :)